_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+
+If the player isn't closed (by using the cross button or swiping down on it), and instead stays minimized, you can tap "Back" to go through those videos in reverse order! You are making history, as the lead role of your own exploration. Pick videos for a quickly built video queue before watching.
+Before this was only possible by using the background or pop-up player.
 
 ### Thumbnail progress bar
 
-You can only see this when you play the video in the background or pop-up, but there is now a shiny new progress bar underneath the thumbnail! It updates in real time, gets a little counter and a red bar - it's great, really.  
+Bar progress, nail a thumb! If playing video in the background or in a pop-up, take note of the shiny progress bar underneath the thumbnail.
+It updates in real time, with a little counter and a red bar. The audacity.
 <img class="no-flow vertical" src="/img/screenshots/progress_bar.png"/>
 
-All of what you've read above was made possible by the lean and mean coding machine, the one and only - (drumroll) - [@avently](https://github.com/avently)! Seriously, you should all sacrifice your first born child in his name. Or just say thank you, whatever works out for ya (that would be kind of boring, though).
+This fine list of changes were made possible by the steadily awesome [@avently](https://github.com/avently). Let him live rent free in your head, as you bask in the fruits of his labour.
 
 * * *
 
-On a serious note, this was a massive, massive change to not only the code, but also the app's workflow, so this PR was tested, reviewed and modified lots of times, over a long period of time, before it was allowed to see the light of day with this release. A huge thank you to you, [@avently](https://github.com/avently), for having the patience and dedication to keep adding features and bug fixes until everyone was satisfied! A different person probably would have gone insane. And a big thank you to the maintainers for doggedly reviewing this gigantic PR repeatedly. Those ~6600 additions and ~5600 removals must have felt intimidating!
+This massive, massive change to not only the code, but also the app's workflow, meant the PR had to be tested, reviewed and modified lots of times, over a long period of time. A huge thank you to [@avently](https://github.com/avently), for having the patience and dedication to keep adding features and bug-fixes until everyone was satisfied! A different person probably would have gone insane. And a big thank you to the maintainers for doggedly reviewing this gigantic PR repeatedly. Totalling ~6600 additions and ~5600 removals, the hat is off.
 
-Naturally, since this was such a massive change, several bugs also slipped through. But [@avently](https://github.com/avently), with the help of [@blackbox87](https://github.com/blackbox87), [@Stypox](https://github.com/Stypox) and [@TobiGr](https://github.com/TobiGr) fixed them rapidly as they were reported. 
```

Comment:
There was already "massive, massive change". Now there is "massive change". I am dying a thousand deaths reading that. "fixed them rapidly as they were reported" is an unclear construction.

Reply:
Agreed. We could have used a different word. 'Major', perhaps.

> "fixed them rapidly as they were reported" is an unclear construction.

In what way? Is it ambiguous? Misleading?

Reply:
It can mean two things. Not quite ambiguous.